src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -170,9 +207,23 @@ public void saveCondition() {
 	@Then("the system adds New Condition in Conditions table")
 	public void systemAddsCondition() {
 		assertNotNull(conditionsPage.getConditionsList());
+	}
+
+	@And("a user clicks on the delete button from dashboard")
+	public void deleteCondition() {
+		conditionsPage.deleteFirstActive();
+		conditionsPage.confirmDeleteCondition();
+		assertNotNull(conditionsPage.getConditionsList());
 		dashboardPage = conditionsPage.clickReturn();
```

Comment:
isn't  conditionsPage.clickReturn();`  supposed to be invoked after the system has deleted the condition?